README.md

```diff
@@ -117,14 +117,14 @@ This is the recommended way of building all of the BlazingSQL components and dep
 ```bash
 conda create -n bsql python=$PYTHON_VERSION
 conda activate bsql
-./dependencies.sh 21.06 $CUDA_VERSION
+./dependencies.sh 21.08 $CUDA_VERSION
```

Comment:
this is the part of the README for the stable version, which is only at 21.06, so this should not change here